<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KinectWidget</class>
 <widget class="QWidget" name="KinectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>554</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>993</width>
    <height>554</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>993</width>
    <height>554</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Kinect Control Panel</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	font-family: &quot;Arial&quot;;
	font-size: 10pt;
}

QPushButton {
	min-width: 155px;
	max-width: 155px;
}

QLabel#widgetDepth, #widgetColor, #widgetSkeletons {
	border: 1px solid grey;
}
QLabel#widgetDepth {
	border-bottom: 0px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="mainLayout">
     <property name="spacing">
      <number>9</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="layoutColor">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="labelTitleColor">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Color View</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="widgetColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>640</width>
           <height>480</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>640</width>
           <height>480</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layoutDepth">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelTitleDepth">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Depth and Skeleton View</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="widgetDepth">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>240</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>320</width>
           <height>240</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="widgetSkeletons">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>240</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>320</width>
           <height>240</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutControls">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QLabel" name="labelStatus">
       <property name="indent">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoneRendering">
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Render Tracked Bones</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Render Inferred and Tracked Bones</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAdjustElevation">
       <property name="text">
        <string>Adjust Elevation Angle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonStartStop">
       <property name="text">
        <string>Start Kinect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
